external/concealer/include/concealer/autoware_universe.hpp

```diff
@@ -40,6 +40,8 @@
 #include <tier4_rtc_msgs/msg/cooperate_status_array.hpp>
 #include <tier4_rtc_msgs/srv/cooperate_commands.hpp>
 
+#include "concealer/utility/service_with_validation.h"
```

Comment:
If there is no reason why it must be double-quote, then use angled brackets to match other headers.